### PR TITLE
Avoid as const in js extension

### LINKED ISSUE #166: Compilation error with code generated from protoc-gen-connect-query
With v0.4.3, `protoc-gen-connect-query` generates the following TypeScript (with the plugin option `target=ts`):

```ts
// ...
export const ElizaService = {
  typeName: "connectrpc.eliza.v1.ElizaService",
  methods: {
    // elided
  }
} as const;
// ...
```

With the plugin option `target=js`, the plugin also generates `as const`, which is a syntax error in JavaScript.

### PR DESCRIPTION
**Pull request by paul-sachs**
Fixes #166

### CODE REVIEW

File: packages/protoc-gen-connect-query/src/generateTs.ts [hunk 1]
```diff
@@ -95,11 +95,11 @@ const generateServiceFile =
  */
 export const generateTs: PluginInit["generateJs"] & PluginInit["generateTs"] = (
   schema,
-  extension,
+  extension
```
Comment (timostamm) [inline packages/protoc-gen-connect-query/src/generateTs.ts:98]: Pretty sure prettier wants the comma.

Comment (paul-sachs) [reply packages/protoc-gen-connect-query/src/generateTs.ts:98]: yeah weird, my local copy clearly didn't.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ jobs:
         run: pnpm run generate
 
       - name: Lint
-        run: pnpm run format
+        run: pnpm run format --force
```
[the remark this answers is not in the document]
Comment (paul-sachs) [reply .github/workflows/ci.yaml:47]: Yeah this was a mistake. I'm going to revert that. I was reading the wrong lint failure somehow.

[the remark this answers is not in the document]
Comment (timostamm) [reply .github/workflows/ci.yaml:47]: 👍 my guess is that the diff check update in https://github.com/connectrpc/connect-query-es/pull/165 will clean that up.

**Pull request merged**